sidertests/test_types.py

```diff
@@ -6,6 +6,7 @@
 from .env import session
 from sider import types
 from sider.types import Boolean, ByteString, Date, DateTime, TZDateTime, UUID
+from sider.types import Tuple, Integer, String
```

Comment:
There’s already `from sider.types import ...` line above.  I think these two lines can be merged:

```python
from sider.types import (Boolean, ByteString, Date, DateTime, Integer, String,
                         Tuple, TZDateTime, UUID)
```